### DISCUSSION TITLE
How do I properly write a theme config?

### DISCUSSION THREAD

**Discussion by jonasfroeller** [huntabyte/shadcn-svelte#758] [Help] [answered]
I want to use my Figma theme. The issue is that the colors are all in hex format. While I could convert them to hex, I'd also need to determine the foreground colors and dark-mode colors. Is there a color-theme generator available where I can input my primary, secondary, accent colors, and so forth?

I noticed there's a theme generator at https://www.shadcn-svelte.com/themes, but unfortunately, it's not very useful if you want to use your own colors since you can't input them.

Something like https://themes.fkaya.dev, but with more options.

**Comment by shyakadavis** [comment, marked as answer]
Hi; I've seen a few generators, you could try any of these. (I'll update this reply if I remember others)

1. https://palette-generator.significa.co/
2. https://ui.jln.dev/
3. https://ui-colorgen.vercel.app/

P.S. Please let me know which works best for you.

**Reply by jonasfroeller** [reply]
https://ui.jln.dev works really well, thank you!
